install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -126,7 +126,7 @@ It may not be immediately obvious which SCC was matched by a pod, so the command
 above can be very useful in understanding the UID, supplemental groups, and
 SELinux relabeling in a live container.
 
-Any SCC with a strategy set to *RunAsAny* allows arbitrary values for that
+Any SCC with a strategy set to *RunAsAny* allows specific values for that
```

Comment:
I don't understand this change.  Is it intentional?  To me, "run as any" and "arbitrary" sound like a better match than "run as any" and "specific".